nanome/_internal/_plugin_instance.py

```diff
@@ -63,7 +80,7 @@ def _on_stop(self):
         except:
             Logs.error("Error in on_stop function:", traceback.format_exc())
 
-    def _run(self):
+    def _update(self):
```

Comment:
Doesn't feel like "update" accurately describes either of these methods.
This one is some sort of start up code and the other one is an everything-in-one async thing.

Reply:
Yeah, maybe `update_loop` is more accurate?

Reply:
but the one here isn't an update loop right? It just calls Start

Reply:
expand the collapsed lines, this function and the one from `_plugin_instance_async` are the exact same other than the method used to sleep